lib/provider/volume_manager.go

```diff
@@ -26,6 +26,9 @@ type VolumeManager interface {
 	Type() VolumeType
 
 	// Volume operations
+	// Get the volume profile by using profile name
+	GetVolumeProfileByName(name string) error
```

Comment:
Can you return profile details as well

Reply:
We can change method name but returning profile details is of no use for us. We will have introduce models which wont be used by us. Our objective is to check if profile is enabled or not. Do you have any intention on fetch profile and then do some action or just return as a standard ?